pkg/envoy/ingress_translator.go

```diff
@@ -93,6 +94,7 @@ type cluster struct {
 	VirtualHost     string
 	HealthCheckPath string
 	Timeout         time.Duration
+	Weight          uint32
```

Comment:
`Weight` is not initialized and will default to 0, removing the upstream from load-balancing if the annotation is not found.  
I think we can default it to 1 for ingress without the weight annotation.